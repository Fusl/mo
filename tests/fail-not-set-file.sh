#!/usr/bin/env bash

cd "${0%/*}"
unset __NO_SUCH_VAR
../mo --fail-not-set ./fail-not-set-file.template 2>&1

if [[ $? -ne 1 ]]; then
    echo "Did not return 1"
fi
