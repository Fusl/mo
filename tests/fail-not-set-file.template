* {{__NO_SUCH_VAR}}
