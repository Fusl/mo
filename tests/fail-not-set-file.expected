Env variable not set: __NO_SUCH_VAR
